### DISCUSSION TITLE
控制模块能否隐藏

### DISCUSSION THREAD

**Discussion by tbing2020** [GaiZhenbiao/ChuanhuChatGPT#441] [Q&A]
有个问题，内部团队使用这个程序，登录了都能看到API-key、代理等设置，有人随意更改后，程序就用不了了。。。能否设置不登录只能对话，登录了才能进行参数配置？
在哪里能配置这个需求？

**Comment by tbing2020** [comment]
没人能回复么？

**Reply by Gary2018X** [reply]
可以自己加个权限控制 默认api key这些的visible为False 如果是有权限的人就显示 但是gradio对这块支持好像不是太好 需要自己搞一些巧妙的方法做

**Reply by Keldos-Li** [reply]
别急
